manager/src/main/java/com/redhat/service/bridge/manager/models/Processor.java

```diff
@@ -39,7 +39,7 @@
                         "where p.status in (:statuses) " +
                         "and p.bridge.status='READY' " +
                         "and p.shardId=:shardId " +
-                        "and (c is null or c.status = 'READY')"),
+                        "and (c is null or c.status = 'READY' or c.status = 'FAILED')"),
```

Comment:
Does this mean that the shard starts provisioning a processor even if the connector is in a FAILED status?

Reply:
> @lucamolteni How does this overlap with the work that @manstis is doing right now? Are we going to be causing @manstis any headaches if we merge this?

It completely stamps all over it to be honest; but this PR can remain "for now" as a quick fix.

My work fails Processors too; should the Connector fail to be provisioned.

Reply:
It was meant to actually delete processors with failed connectors, but since it should be impossible to have such let me remove the condition